frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -97,21 +116,20 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
         <div className="ceph-add-capacity__modal">
           <div
             className={classNames('ceph-add-capacity__sc-dropdown', {
-              'ceph-add-capacity__sc-dropdown--margin':
-                storageClass?.provisioner !== NO_PROVISIONER,
+              'ceph-add-capacity__sc-dropdown--margin': !isNoProvionerSC,
             })}
           >
             <label className="control-label" htmlFor="storageClass">
               Storage Class
               <FieldLevelHelp>{storageClassTooltip}</FieldLevelHelp>
             </label>
-            <OCSStorageClassDropdown onChange={setStorageClass} />
+            <OCSStorageClassDropdown onChange={onChange} />
```

Comment:
`setStorageClass` can be directly called here. No need to create a separate function.

Reply:
The underlying function definition accepts a function like `onChange`

Reply:
Are you saying that this `onChange` is used at other place too as I don't see any?
I was suggesting to leave this like this(the way it was earlier) - 
`<OCSStorageClassDropdown onChange={setStorageClass} />`

Reply:
I need to change the typings here and look for changes at other places.
I dont want to do refactors in this PR.
https://github.com/openshift/console/blob/2d0cfb2b0b27b65126cf79f4c03aab028223e5b5/frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx#L52